File: app/src/main/java/com/novoda/frankboylan/ncu/nodes/Layer.java
package com.novoda.frankboylan.ncu.nodes;

import java.util.List;

class Layer {

    private int layerNumber;

    private List<NodeLite> nodeList;

    public int getLayerNumber() {
        return layerNumber;
    }

    public void setLayerNumber(int layerNumber) {
        this.layerNumber = layerNumber;
    }

    public List<NodeLite> getNodeList() {
        return nodeList;
    }

    public void setNodeList(List<NodeLite> nodeList) {
        this.nodeList = nodeList;
    }
}

File: app/src/main/java/com/novoda/frankboylan/ncu/ktnodes/Layer.kt
package com.novoda.frankboylan.ncu.ktnodes

import com.google.gson.annotations.SerializedName

data class Layer(
        @SerializedName("layer-number") val layerNumber: Int,
        @SerializedName("nodes") val nodeLiteList: List<NodeLite>
)
